examples/jupyter_lab.yaml

```diff
@@ -0,0 +1,7 @@
+resources:
```

Comment:
Might be good to add a header docstr, e.g.,
```
# Example: Launch Jupyter Lab and auto-expose its port to Internet.
#
# Usage:
#     $ sky launch -c jupyter jupyter_lab.yaml
#     # Then open browser ....?
```
Actually, writing here makes me realize how do we plan to connect to `jupyter:29324` without port-forwarding?

Reply:
Currently I just use the head IP printed by `sky status -a` and manually replace it

Reply:
It will output something like 
```bash
(task, pid=24010) [I 2023-08-02 02:16:39.369 ServerApp] Jupyter Server 2.7.0 is running at:
(task, pid=24010) [I 2023-08-02 02:16:39.369 ServerApp] http://ip-172-31-87-123:29324/lab?token=<token>
(task, pid=24010) [I 2023-08-02 02:16:39.369 ServerApp]     http://127.0.0.1:29324/lab?token=<token>
```
and just replace the `127.0.0.1` with head IP